### PR TITLE
testcase: Adds new unit test case for resource alicloud_resource_manager_shared_target alicloud_resource_manager_shared_resource alicloud_resource_manager_role

### PR DESCRIPTION
**Pull request by thisnihonglei**

### CODE REVIEW

File: alicloud/resource_alicloud_resource_manager_role_test.go [hunk 1]
```diff
@@ -94,3 +103,260 @@ func ResourceManagerRoleBasicdependence(name string) string {
 	data "alicloud_account" "default" {}
 `)
 }
+
+func TestUnitAlicloudResourceManagerRole(t *testing.T) {
```
Comment (thisnihonglei) [inline alicloud/resource_alicloud_resource_manager_role_test.go:107]: 单测覆盖率94.9%

File: alicloud/resource_alicloud_resource_manager_shared_resource_test.go [hunk 2]
```diff
@@ -81,3 +90,220 @@ resource "alicloud_resource_manager_resource_share" "default" {
 }
 `, name)
 }
+
+func TestUnitAlicloudResourceManagerSharedResource(t *testing.T) {
```
Comment (thisnihonglei) [inline alicloud/resource_alicloud_resource_manager_shared_resource_test.go:94]: 单测覆盖率94.6%

File: alicloud/resource_alicloud_resource_manager_shared_target_test.go [hunk 3]
```diff
@@ -70,3 +79,217 @@ resource "alicloud_resource_manager_resource_share" "default" {
 
 `, name)
 }
+
+func TestUniAlicloudResourceManagerSharedTarget(t *testing.T) {
```
Comment (thisnihonglei) [inline alicloud/resource_alicloud_resource_manager_shared_target_test.go:83]: 单测覆盖率94.4%

File: website/docs/r/resource_manager_role.html.markdown [hunk 4]
```diff
@@ -52,7 +52,7 @@ The following arguments are supported:
 
 The following attributes are exported:
 
-* `id` - This ID of Resource Manager role. The value is set to `role_name`.
```
Comment (xiaozhu36) [inline website/docs/r/resource_manager_role.html.markdown:55]: id 也是有的啊

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**